src/BTVNanoCommissioning/workflows/ctag_Wc_valid_sf.py

```diff
@@ -140,30 +140,13 @@ def process_shift(self, events, shift_name):
         ## Soft Muon cuts
         soft_muon = events.Muon[softmu_mask(events, self._campaign)]
         req_softmu = ak.count(soft_muon.pt, axis=1) >= 1
-        soft_muon = ak.pad_none(soft_muon, 1, axis=1)
-
-        ## Muon-jet cuts
-        mu_jet = event_jet[
-            (
-                ak.all(
-                    event_jet.metric_table(soft_muon) <= 0.4, axis=2, mask_identity=True
-                )
-            )
+        mujetsel = (
+            (ak.all(event_jet.metric_table(soft_muon) <= 0.4, axis=2))
```

Comment:
The `mask_identity=True` should be there to avoid `None`  events pass `True` criteria. 
 The `ak.fill_none(mujetsel,False,axis=-1)` then remove `None`, this cut can be proceed anywhere. (Then you don't need `mujetsel2`)